gtests/net/mptcp/dss/mpc_with_data_server_window_close.pkt

```diff
@@ -0,0 +1,31 @@
+// connection initiated by packetdrill
+--tolerance_usecs=100000
+`../common/defaults.sh`
+
++0     socket(..., SOCK_STREAM, IPPROTO_MPTCP) = 3
++0     setsockopt(3, SOL_SOCKET, SO_REUSEADDR, [1], 4) = 0
+
++0     `nstat -n`
++0     bind(3, ..., ...) = 0
++0     listen(3, 1) = 0
++0       <  S   0:0(0)                  win 32792  <mss 1460, sackOK, nop, nop, nop, wscale 7, mpcapable v1 flags[flag_h] nokey>
++0       >  S.  0:0(0)        ack 1                <mss 1460, nop, nop, sackOK, nop, wscale 8, mpcapable v1 flags[flag_h] key[skey]>
++0.01    <   .  1:1(0)        ack 1     win 10                                               <mpcapable v1 flags[flag_h] key[ckey=2, skey]>
++0     accept(3, ..., ...) = 4
+
++0.01    <  P.  1:101(100)    ack 1     win 10                                               <mpcapable v1 flags[flag_h] key[ckey=2, skey] mpcdatalen 100, nop, nop>
++0.0     >   .  1:1(0)        ack 101              <dss dack8=101  nocs>
+
+// fill the receiver window and get zero window ack
++0     write(4, ..., 2000) = 2000
++0       >  P.  1:1281(1280)  ack 101              <dss dack8=101  dsn8=1    ssn=1 dll=1280 nocs, nop, nop>
++0       <   .  101:101(0)    ack 1281  win 0      <dss dack8=1281 nocs>
+
+// zero window probe will trigger; note the MPTCP-level data reinjection used
+// to trigger the probe
++0.3     >   .  1280:1280(0)  ack 101              <dss dack8=101  nocs>
++0       <   .  101:101(0)    ack 1281  win 225    <dss dack8=1281 nocs>
++0       >  P.  1281:1282(1)  ack 101              <dss dack8=101  dsn8=1280 ssn=1281 dll=1    nocs, nop, nop>
+
+// neagle takes action, this one is a little bit delayed
++0.3       >  P.  1282:2002(720) ack 101             <dss dack8=101  dsn8=1281 ssn=1282 dll=720  nocs, nop, nop>
```

Comment:
(details: some columns are not aligned, but I will fix that when applying the patches.)